src/tools/file-base.r

```diff
@@ -206,6 +206,8 @@ modules: [
     UUID ../extensions/uuid/mod-uuid.c [
         ;if Linux
     ]
+
+    ODBC ../extensions/odbc/mod-odbc.c []
 ]
```

Comment:
Modules are not defined in this file any more, I should have delete these lines. They are now in [%src/tool/make-make.r](https://github.com/metaeducation/ren-c/blob/master/src/tools/make-make.r#L455), where you can specify all libraries and flags they need programmingly.